theano/tensor/tests/test_basic.py

```diff
@@ -1677,6 +1677,8 @@ def _numpy_true_div(x, y):
     expected_gammaln = scipy.special.gammaln
     expected_psi = scipy.special.psi
     expected_chi2sf = lambda x, df: scipy.stats.chi2.sf(x, df).astype(x.dtype)
+    expected_j0 = scipy.special.j0
+    expected_j1 = scipy.special.j1
```

Comment:
You must add those variable in the else close around line 1700. Otherwise, there will be a crash is scipy isn't installed.